extensions/azurecore/src/account-provider/utils/fileDatabase.ts

```diff
@@ -91,7 +91,7 @@ export class FileDatabase {
 
 	public async initialize(): Promise<void> {
 		this.isInitialized = true;
-		this.setupSaveTask();
+		this.saveInterval = setInterval(() => this.save(), 20 * 1000);
```

Comment:
This was actually a bug here - we were never assigning the saveInterval so shutdown would never call it. 

(but we only call shutdown during tests so didn't have any actual runtime issues)